### PR TITLE
Readme, Flame animation

### PR DESCRIPTION
**Pull request by Woflje**

### CODE REVIEW

**Comment by Woflje** [comment]
barber takes multiple colours as arguments, how should that be handled properly in yaml to work with the managing?
```yaml
parameters:
  speed: 1
  update_rate: 20
  colours:
    - [1, 0, 0]   # Red
    - [1, 1, 1]   # White
    - [0, 0, 1]   # Blue
```

**Comment by MaxWinsemius** [comment]
> barber takes multiple colours as arguments, how should that be handled properly in yaml to work with the managing?
> 
> ```yaml
> parameters:
>   speed: 1
>   update_rate: 20
>   colours:
>     - [1, 0, 0]   # Red
>     - [1, 1, 1]   # White
>     - [0, 0, 1]   # Blue
> ```

Hmm good one, I think this would then be managed with something like

`./manage animation start barber <interface> --option colours=1;0;0,1;1;1,0;0;1`

I have to make a distinguishable splitter in a single array (I don't want to play too much with allowing `[]` as they fiddle around with bash and parsing is harder).

**Comment by Woflje** [comment]
> > barber takes multiple colours as arguments, how should that be handled properly in yaml to work with the managing?
> > ```yaml
> > parameters:
> >   speed: 1
> >   update_rate: 20
> >   colours:
> >     - [1, 0, 0]   # Red
> >     - [1, 1, 1]   # White
> >     - [0, 0, 1]   # Blue
> > ```
> 
> Hmm good one, I think this would then be managed with something like
> 
> `./manage animation start barber <interface> --option colours=1;0;0,1;1;1,0;0;1`
> 
> I have to make a distinguishable splitter in a single array (I don't want to play too much with allowing `[]` as they fiddle around with bash and parsing is harder).

Sounds fine

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+In `src/terminal_testbed` run
+
+```bash
+g++ -o a.out ./main.cpp
+```
+
+## Usage
+
+1. Run `boot.sh`
+2. Run `start_testbed.sh`
```
Comment (MaxWinsemius) [inline README.md:21]: Note that except for line 20 it's only applicable for the testbed. Other than `boot` you don't need to run anything if you're just handling non-testbed interfaces.

Not something to change for this PR, but might be a nice modification later

File: src/animations/barber.py
Comment (MaxWinsemius) [inline src/animations/barber.py:1]: I don't know exactly why, but it seems to be broken on the first 14 leds. But it works other than that. Might break on other configurations as well, but let's not look at that :P

**Review by MaxWinsemius — APPROVED** [review]
Other than the comments I made, LGTM.

**Pull request merged**